includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -316,7 +316,8 @@ public static function subscription_pricing_fields() {
 				?>
 			</label>
 			<span class="wrap">
-				<input type="text" id="_subscription_price" name="_subscription_price" class="wc_input_price wc_input_subscription_price" placeholder="<?php echo esc_attr_x( 'e.g. 5.90', 'example price', 'woocommerce-subscriptions' ); ?>" step="any" min="0" value="<?php echo esc_attr( wc_format_localized_price( $chosen_price ) ); ?>" />
+				<?php // Translators: %s: formatted example price value. ?>
+				<input type="text" id="_subscription_price" name="_subscription_price" class="wc_input_price wc_input_subscription_price" placeholder="<?php printf( esc_attr_x( 'e.g. %s', 'example price', 'woocommerce-subscriptions' ), esc_attr( wc_format_localized_price( '5.90' ) ) ); ?>" step="any" min="0" value="<?php echo esc_attr( wc_format_localized_price( $chosen_price ) ); ?>" />
```

Comment:
This is certainly a nit, so feel free to skip, but I wonder if this would be very marginally better (that is, pass the final result through `esc_attr()` *once* instead of passing each component through separately):

```suggestion
				<input type="text" id="_subscription_price" name="_subscription_price" class="wc_input_price wc_input_subscription_price" placeholder="<?php echo esc_attr( sprintf( _x( 'e.g. %s', 'example price', 'woocommerce-subscriptions' ), wc_format_localized_price( '5.90' ) ) ); ?>" step="any" min="0" value="<?php echo esc_attr( wc_format_localized_price( $chosen_price ) ); ?>" />
```

Reply:
Fixed on https://github.com/Automattic/woocommerce-subscriptions-core/pull/807/commits/ce3bdaf566b50a9696525c1248b0b4e287b93dc8.